PDE/advection1.m

```diff
@@ -1,7 +1,7 @@
 function pde = advection1
 % 1D test case using continuity equation, i.e., 
 %
-% df/dt == -df/dx
+% df/dt == -df/dx + source(x)
```

Comment:
Let's change this comment to be the actual equation we are solving, i.e., with the `df/dt == -2*df/dx -2*sin(x)`

Reply:
The advection1.m comment has been changed to the equation we are solving.